airflow/cli/commands/celery_command.py

```diff
@@ -18,13 +18,16 @@
 """Celery command."""
 from __future__ import annotations
 
+import logging
 from contextlib import contextmanager
 from multiprocessing import Process
+import sys.stderr, sys.stdout
```

Comment:
I believe accord to pep8 it should be just "import sys".
In the code you still should use it as sys.stderr/sys.stdout

Reply:
I was always trying to keep imports as narrow as possible, but I am ok with that change.
Updated.